Postman/Visual Programming.postman_collection.json

```diff
@@ -291,6 +291,25 @@
 				}
 			},
 			"response": []
+		},
+		{
```

Comment:
I had separated some of the Postman tests into two collections in #29 for Workflow and Nodes, so I think there's going to be a merge conflict here. I think you have a better handle on the testing than I do, so I would defer to your judgement on how best to approach this both now and going forward.

Reply:
Yep, I think that distinction is good. I've added the new endpoint to the Visual Programming Nodes collection.